misc/images/jobs/Dockerfile

```diff
@@ -11,7 +11,7 @@ MZFROM ubuntu-base
 
 RUN apt-get update && apt-get -qy install ca-certificates
 
-COPY examples/persistcli /usr/local/bin/persistcli
+COPY persistcli /usr/local/bin/persistcli
```

Comment:
Is this covered by CI?

Reply:
Yup. That's how I caught it, in fact. Also verified via a `./bin/mzimage run --dev jobs -- --help`